packages/launch/src/launch/index.tsx

```diff
@@ -21,26 +21,16 @@ const FocusedLaunchModal: React.FunctionComponent< Props > = ( { onClose } ) =>
 			overlayClassName="launch__focused-modal-overlay"
 			bodyOpenClassName="has-focused-launch-modal"
 			onRequestClose={ onClose }
-			title=""
+			title={ __( 'Complete setup' ) }
+			// remove the wrong type when https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49156 is merged
```

Comment:
Looks like it is merged. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49156